<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scoreDlg</class>
 <widget class="QDialog" name="scoreDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/new/prefix1/pic/njupt.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="scoreTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>平时成绩</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>半期考试</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>期末考试</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总评成绩</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>等级</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>所有考生考试情况总览</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>班级总评平均分：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>及格率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="aveLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
  <widget class="QLabel" name="passRate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>90%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/new/prefix1/pic/background_4.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>scoreTable</zorder>
  <zorder>label</zorder>
  <zorder>line_2</zorder>
  <zorder>line</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>aveLabel</zorder>
  <zorder>passRate</zorder>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
